dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -295,6 +305,16 @@ public void createRelationshipWriteAccessRightItem() throws Exception {
                                               .andExpect(status().isCreated())
                                               .andReturn();
 
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
please see
https://github.com/DSpace/DSpace/blob/d25463fedb2de393abef8d65b685a5a3e3408762/dspace-spring-rest/src/test/java/org/dspace/app/rest/WorkflowItemRestRepositoryIT.java#L708

to simplify the above code and to assure that any object created via REST is properly deleted also in case of test failure